config/config.go

```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
+	"github.com/prometheus/alertmanager/pkg/labels"
```

Comment:
Let's put this into a third block further down. That's how we usually do it with packages from the same repository (see https://github.com/prometheus/alertmanager/blob/master/cmd/alertmanager/main.go ).